.github/workflows/CI.yml

```
@@ -61,10 +61,44 @@ jobs:
           path-to-lcov: lcov.info
 
   latex:
-    name: "PDF/LaTeX backend"
-    runs-on: ubuntu-latest
+    name: "PDF/LaTeX backend - ${{ matrix.os }}"
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - ubuntu-latest
+          - windows-latest
     steps:
       - uses: actions/checkout@v4
+      - name: "Download and install DejaVu fonts"
+        run: |
+          Invoke-WebRequest "https://github.com/dejavu-fonts/dejavu-fonts/releases/download/version_2_37/dejavu-fonts-ttf-2.37.zip" -OutFile "DejaVu.zip"
+          Expand-Archive -Path "DejaVu.zip" -DestinationPath "."
+          $fonts = (New-Object -ComObject Shell.Application).Namespace(0x14)
+          foreach ($file in gci -Path ".\dejavu-fonts-ttf-2.37\ttf" "*.ttf")
+          {
+            $fonts.CopyHere($file.fullname)
+          }
+        if: matrix.os == 'windows-latest'
+      - uses: actions/setup-python@v5
+        with:
+          python-version: "3.13"
+        if: matrix.os == 'windows-latest'
+      - name: "install pygments for tectonic"
+        run: pip install pygments
+        if: matrix.os == 'windows-latest'
+      - uses: teatimeguest/setup-texlive-action@v3
+        with:
+          packages: |
+            scheme-medium
+            hyperref
+            minted
+            newunicodechar
+            transparent
+            upquote
+            tabulary
+            tcolorbox
```

Can we also only do this step when running in Windows?

```suggestion
            tcolorbox
        if: matrix.os == 'windows-latest'
```

---

@FriesischScott I did the change in [bd3359e](https://github.com/JuliaDocs/Documenter.jl/pull/2708/commits/bd3359eecb86a8f4371959c639a47586afa5e3b8), but still would like to double check that this seems correct to you too?

---

Only installing `texlive` on Windows breaks the native test on `ubuntu-latest`. I think it's worth doing it to ensure the native build runs on both Windows and Linux.

---

I went ahead and reverted the commit. If this is about the time it takes to install texlive I could find out what the smallest set of necessary packages is starting from `scheme-basic` instead of `scheme-medium`.

---

Sounds good, thank you! I just wasn't sure if it was necessary and intentionally enabled for Linux or not.